### DISCUSSION TITLE
NTM resistance mutation database and drug resistance prediction.

### DISCUSSION THREAD

**Discussion by harrismia** [jodyphelan/NTM-Profiler#6] [Ideas]
It would be great to add resistance prediction capabilities similar to what is available for TB Profiler.   Is there a known database of resistance mutations for NTM?

**Comment by jodyphelan** [comment]
From what I have seen it is mostly scattered in the literature (e.g. https://www.ncbi.nlm.nih.gov/pmc/articles/PMC3028756/). I haven't come across a central database, though I could have missed it. I'm happy to make a start for some of the most clinically relevant NTMs.

**Comment by harrismia** [comment]
That would be great! It might be good to have a similar separate database (similar to tbdb for TB Profiler) that can be updated as new variants are reported in the literature.     ntmdb ?     I guess this would also need to include the name of the NTM ?

**Comment by harrismia** [comment]
One more publication:  [https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6325232/](url)

**Comment by jodyphelan** [comment]
Resistance prediction is now available for _Mycobacteroides abscessus_ in v0.1.1

**Reply by harrismia** [reply]
Fantastic! Can't wait to give it a try!

**Comment by harrismia** [comment]
Other suggested organisms are:

Mycobacterium avium complex (MAC)
Mycobacterium kansasii

**Comment by yugen-miyahara** [comment]
I would like to suggest adding resistance prediction for: Mycolicibacterium fortuitum

**Comment by harrismia2** [comment]
Other suggested organisms: 

Mycobacterium xenopi
Mycobacterium chelonae

**Comment by harrismia2** [comment]
This article is interesting [Plasmid-mediated drug resistance in mycobacteria: the tip of the iceberg?](https://doi.org/10.1128/jcm.00628-23).   Is there a way to computationally look for plasmids carrying erm?

**Comment by sachiwije** [comment]
I am wondering if it is possible to include _Mycobacterium ulcerans_

**Comment by pmenzel** [comment]
Mycolicibacterium agri could also be added for species detection: https://www.ncbi.nlm.nih.gov/datasets/genome/GCF_010722915.1/

**Reply by jodyphelan** [reply]
_M. agri_ is already added to the species database. Are you finding it does not detect?
https://pathogen-profiler.github.io/ntm-db/species-detection/

**Reply by pmenzel** [reply]
sorry, I missed that is included already!
